README.md

```diff
@@ -1,77 +1,82 @@
+- You have registered (created) a Repeater in the Bright app and copied the generated Repeater ID. For the instructions on how to register a Repeater, see [here](https://docs.brightsec.com/docs/manage-repeaters#create-a-new-repeater).
+- You have copied the Bright Project ID under which you want to run a scan. A Project ID is required. If you do not have any custom projects, use the Default Project ID.
 
-#### 1. Install NexPloit CLI globally
+#### 1. Install Bright CLI globally
 
 ```bash
-npm install @neuralegion/nexploit-cli -g
+npm install @brightsec/cli -g
 ```
 
-You can validate the installation by going to the directory of your project and running the command:
+You can make sure the installation worked by executing the following command:
 
 ```bash
-nexploit-cli -h
+bright-cli --version
 ```
 
-This will show you a list of possible commands for NexPloit CLI, for a full list go [here](https://docs.brightsec.com/docs/command-list)
+It should return the latest Bright CLI version.
 
 #### 2. Activate the Repeater
 
 ```bash
-nexploit-cli repeater \
+bright-cli repeater \
   --token {TOKEN} \
-  --id {ID} \
-  --bus amqps://amq.app.brightsec.com:5672
+  --id {ID}
 ```
 
 #### 3. Start a new scan with a Crawler
 
 ```bash
-nexploit-cli scan:run \
+bright-cli scan:run \
   --token {TOKEN} \
   --repeater {ID} \
-  --name "My First Scan" \
-  --crawler https://www.example.com \
+  --name "Bright scan" \
+  --crawler {TARGET_URL} \
+  --project {PROJECT_ID}  \ #If you do not have any custom projects, specify the Default Project ID.
   --smart
 ```
 
-This command will initialize a new scan engine in the cloud, which will start scanning the target via the local [Repeater](https://docs.brightsec.com/docs/on-premises-repeater-local-agent).
+This command will initialize a new scan engine on the cloud, which will start scanning the target in the Repeater mode.
+
+We recommend that you use the `--smart` option to optimize the scan coverage and time. This enables you to use automatic smart decisions, such as parameter skipping, detection phases and so on.
 
 #### 4. Check out the scan results
 
-You can follow the scan status here: https://app.brightsec.com/scans, or by using the [NexPloit CLI polling](https://docs.brightsec.com/docs/checking-scan-status) command.
+You can follow the scan status in the [Bright app](https://app.brightsec.com/scans) or by using the [**Bright CLI** polling](https://docs.brightsec.com/docs/checking-scan-status) command.
 
 ## 📚 Full Documentation
 
-**NexPloit CLI** can do so much more! A full documentation with usage examples is available on [Bright's knowledgebase](https://docs.brightsec.com)
+The **Bright CLI** can do so much more! You can find a full guide with the usage examples in the [Bright docs](https://docs.brightsec.com/docs/getting-started-with-bright-cli).
 
 ## 📝 License
 
-Copyright © 2021 [Bright](https://github.com/NeuraLegion).
+Copyright © 2022 Bright Security Inc.
```

Comment:
Please update the copyright in the LICENSE file as well

```suggestion
Copyright © 2023 [Bright Security Inc.](https://brightsec.com/)
```

Reply:
@aborovsky you have to change the year in LICENSE file https://github.com/NeuraLegion/nexploit-cli/blob/c23b1d872da42cd9e5f4cf918152269540dacf01/LICENSE#L3-L2